.github/workflows/stale.yml

```
@@ -22,3 +22,28 @@ jobs:
           stale-pr-message: |
             This issue has been automatically marked as stale because it has not had recent activity. It will be
             closed if no further activity occurs. Thank you for your contributions.
+  stale-good-first-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          days-before-stale: 21
+          stale-issue-message: |
+            This issue will be unassigned in a week if no further activity is seen.
+            If you are active please provide an update on the status of the issue and if you would like to continue
+            working on it.
+          include-only-assigned: true
+          remove-stale-when-updated: true
+          only-issue-labels: 'good first issue'
+
+  to-be-unassigned-label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          days-before-stale: 28
+          stale-issue-message: |
+            This issue will be labelled 'to be unassigned' because no response has been encountered.
+          include-only-assigned: true
+          only-issue-labels: 'good first issue'
+          stale-issue-label: 'to be unassigned'
```

It would be better to put those steps into one job so that they run sequentially.

```suggestion

  stale-good-first-issues:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/stale@v8
        with:
          days-before-stale: 21
          stale-issue-message: |
            This issue will be unassigned in a week if no further activity is seen.
            If you are active please provide an update on the status of the issue and if you would like to continue
            working on it.
          include-only-assigned: true
          remove-stale-when-updated: true
          only-issue-labels: 'good first issue'

      - uses: actions/stale@v8
        with:
          days-before-stale: 28
          stale-issue-message: |
            This issue will be labelled 'to be unassigned' because no response has been encountered.
          include-only-assigned: true
          only-issue-labels: 'good first issue'
          stale-issue-label: 'to be unassigned'
```

---

Should I add the changes as well as remove the the spaces in line 39.